script.module.clouddrive.common/clouddrive/common/ui/addon.py

```diff
@@ -95,6 +96,7 @@ def __init__(self):
         self._system_monitor = KodiUtils.get_system_monitor()
         self._account_manager = AccountManager(self._profile_path)
         self._pin_dialog = None
+        self.iskrypton = KodiUtils.get_home_property('iskrypton') == 'true'
```

Comment:
Please use our api

Reply:
I'll change this.

Reply:
Ok. I took a look and this is the thing: 
I'm actually using the api to set that property. 
I need it like this because I'm using this property to set the visibility of a setting:
`visible="!String.IsEmpty(window(home).property(iskrypton))"`
See the settings.xml here: https://github.com/xbmc/repo-plugins/pull/2052/files

Some settings will be visible only for krypton and others won't. 

I tried to read and use an info label directly there, but the condition `String.StartsWith(System.BuildVersion,17.)` never worked.
So I had to set that property, which uses the same infolabel and finally worked.